templates/.env-router

```diff
@@ -1,14 +1,37 @@
+# Default Helium's seed nodes
 ROUTER_SEED_NODES=/ip4/35.166.211.46/tcp/2154,/ip4/44.236.95.167/tcp/2154
-ROUTER_CONSOLE_ENDPOINT=http://console:4000
-ROUTER_CONSOLE_WS_ENDPOINT=ws://console:4000/socket/router/websocket
-ROUTER_CONSOLE_SECRET=[must match ROUTER_SECRETS from .env]
-ROUTER_OUI=3
-ROUTER_DEFAULT_DEVADDR=AAQASA==
-ROUTER_SC_OPEN_DC_AMOUNT=100000
-ROUTER_SC_EXPIRATION_INTERVAL=45
+
+# OUI used by router (see https://developer.helium.com/blockchain/blockchain-cli#oui)
+ROUTER_OUI=999
+
+# State Channel Open amount
+ROUTER_SC_OPEN_DC_AMOUNT=2000
+
+# State Channel block expiration
+# FYI: There is a cost to open a state channel (about 35k DC)
+# Current min: 15 blocks, max abbout 5000 blocks
+ROUTER_SC_EXPIRATION_INTERVAL=120
+
+# State Channel Expiration Buffer
+# Minimum distance state channels can expire within another expiration
+# default: 15 when not set
+ROUTER_SC_EXPIRATION_BUFFER=15
+
+# Console's connection info (see https://github.com/helium/console)
+ROUTER_CONSOLE_ENDPOINT=http://helium_console:4000
+ROUTER_CONSOLE_WS_ENDPOINT=ws://helium_console:4000/socket/router/websocket
+# Public facing endpoint to POST downlinks for devices
+ROUTER_CONSOLE_DOWNLINK_ENDPOINT=http://helium_console:4000
+ROUTER_CONSOLE_SECRET=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
not [must match ROUTER_SECRETS from .env]?

Reply:
```suggestion
ROUTER_CONSOLE_SECRET=[must match ROUTER_SECRETS from .env]
```